homeassistant/components/image_processing/opencv.py

```diff
@@ -7,22 +7,56 @@
 from datetime import timedelta
 import logging
 
+import requests
+
+import voluptuous as vol
+
 from homeassistant.core import split_entity_id
 from homeassistant.components.image_processing import (
-    ImageProcessingEntity, PLATFORM_SCHEMA)
-from homeassistant.components.opencv import (
-    ATTR_MATCHES, CLASSIFIER_GROUP_CONFIG, CONF_CLASSIFIER, CONF_ENTITY_ID,
-    CONF_NAME, process_image)
+    CONF_SOURCE, CONF_ENTITY_ID, CONF_NAME, PLATFORM_SCHEMA,
+    ImageProcessingEntity)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['opencv-python==3.2.0.6', 'numpy==1.12.0']
```

Comment:
and I'm curious as to what the module would be for compiled opencv. Ideas?

I see the core has some pip setup methods - but it only seems to process the `REQUIREMENTS` field, perhaps I can invoke that with a yield if the module isn't imported?

The opencv-python 3.2.0.7 has some better handling of architectures - I'll see if it works on pi tonight.